### ISSUE TITLE
Trips Example with babel loader

### ISSUE THREAD

**Issue by anthonyemberley** [uber/deck.gl#565]
I'm trying to add the TripsLayer to a project that was created with create-react-app. I have it configured where I can display a MapGL component and even the hello world DeckGL Line Layer. However, when I try to include TripsLayer, it complains about loaders:
`Unexpected token (19:3)
You may need an appropriate loader to handle this file type.
SyntaxError: Unexpected token (19:3)`
This happened while compiling the deckgl-overlay.js file. Is there any way to compile this using babel instead of buble-loader?

**Comment by ericsoco** [comment]
Certainly. Can you push your code to a repo so we can check it out? /cc @heshan0131

**Comment by anthonyemberley** [comment]
Thanks for the quick response! Here is the github [link]https://github.com/rijupahwa/nodalWebApp)
The files are all in the map folder

**Comment by ericsoco** [comment]
Private repo?

**Comment by anthonyemberley** [comment]
My apologies, this should work: https://github.com/anthonyemberley/NodalTest

**Comment by ericsoco** [comment]
Hm, not sure why you're seeing a problem.
In your repo, I did this:

```
cd map/trips
npm install
npm start
```

And the trips example compiles and runs fine:
![screen shot 2017-04-11 at 2 49 59 pm](https://cloud.githubusercontent.com/assets/1127259/24932388/326a18a0-1ec6-11e7-884a-943f3c03941f.png)

I'm on node v6.9.4, in case that makes a difference.

Are you attempting to build/run a different folder?

**Comment by anthonyemberley** [comment]
I was trying to copy the code used in map/trips/app.js into the map/src/App.js file so I wouldn't have a node project inside of the main node project. I figured that might be bad practice to have two servers running in order to run the full web app but I'm also pretty new to React / Node so maybe that would be fine?

**Comment by ericsoco** [comment]
I wasn't able to build your repo. Your build script is too long for me to dig into, but running your build failed on the first `import` statement, which tells me you're not transpiling your ES6 modules correctly.

In regards to your initial error, perhaps [this SO link](http://stackoverflow.com/questions/33469929/you-may-need-an-appropriate-loader-to-handle-this-file-type-with-webpack-and-b) helps?

**Comment by anthonyemberley** [comment]
Great I will look into it. Thank you for the help!

**Comment by ibgreen** [comment]
@anthonyemberley Did you ever get this working? I just took a quick peek at your repo. I could be wrong but my assumption is that you didn't include any ES6 transforms in your babel config, which could be consistent with your error messages. 

In your package.json I only see [babel-preset-react-app](https://github.com/facebookincubator/create-react-app/tree/master/packages/babel-preset-react-app). I looked at the code and docs for that preset, and it only includes JSX and object spread support, but not ES6 (e.g. babel-preset-es2015).
